### ISSUE TITLE
Webview in relayer overrides status bar color

### ISSUE THREAD

**Issue by adamjuhasz** [magiclabs/magic-js#434]
### ✅ Prerequisites

- [x] Did you perform a cursory search of open issues? Is this bug already reported elsewhere?
- [x] Are you running the latest SDK version?
- [x] Are you reporting to the correct repository (`magic-sdk`)?

### 🐛 Description

When the relayer is rendered in a view, it will change the status bar color to light.

### 🧩 Steps to Reproduce

1. Set the Status Bar color
2. Render a relayer
3. Status Bar is changed to "light"

### 🤔 Expected behavior

The relayer to not change the status bar color.

### 😮 Actual behavior

Even if we set the status bar color, the rendered WebView takes control of the status bar. Setting `autoManageStatusBarEnabled` to `false` disables WebView from controlling the status bar color. We have tested this prop + setting in a patched version and the status bar is no longer changed.

docs: https://github.com/react-native-webview/react-native-webview/blob/master/docs/Reference.md#autoManageStatusBarEnabled

### 💻 Code Sample

Code segment that is affected: 

https://github.com/magiclabs/magic-js/blob/7d634c0184f7daf464b7607cad6317c20bff0a0a/packages/%40magic-sdk/react-native-expo/src/react-native-webview-controller.tsx#L125-L130

### 🌎 Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| `@magic-sdk/react-native-expo`     | 14 |
| Browser          | Expo 47 |
| `yarn`           | npm |
| Operating System | iOS 16 |

**Comment by adamjuhasz** [comment]
Happy to make a PR for this if that speeds up resolution?

**Comment by Ariflo** [comment]
Thanks for offering @adamjuhasz, we're working on sending up a fix before the EOD today.

**Comment by Ariflo** [comment]
@adamjuhasz we're having troubleshooting reproducing the error on our end:can you confirm how it is you are styling `<StatusBar />` exactly?

**Comment by Ariflo** [comment]
@adamjuhasz we're having troubleshooting reproducing the error on our end:

- Can you confirm how it is you are styling `<StatusBar />` exactly?
- Can you provide any screenshot or recording of the error?
- Are you using `expo-status-bar` or React Native's StatusBar?

Feel free to provide any additional information you think will help us get to the bottom of this one. Thanks!

**Comment by adamjuhasz** [comment]
1 & 3 We are using it from `expo-status-bar` with a representative code example. We also tried it the RN's StatusBar and had the same result.

Usage Example
```
import { StatusBar } from "expo-status-bar";
import { Magic } from "@magic-sdk/react-native-expo";

const MagicRelayer = magic.Relayer;


function App(): JSX.Elememt => {
  return (
    <>
       <MagicRelayer />
       <StatusBar style={"light"} />

       <NavigationContainer />
    </>);
}
```

Video showing `MagicRelayer` re-setting the color from what we manually set (white) to WebView's default black during a hot-reload re-render. If we remove `MagicRelayer` from the render tree the color stays white.
 
https://user-images.githubusercontent.com/757940/213828601-02ef01fa-393a-415d-861a-f85541346b19.mp4

**Comment by Ariflo** [comment]
@adamjuhasz  Thanks for the speedy reply and your patience here! 

A [PR](https://github.com/magiclabs/magic-js/pull/436) with your recommended fix is up and should be merged soon. In the meantime, assuming this blocks you, feel free to install the canary version with the fix implemented `yarn add @magic-sdk/react-native-expo@14.0.1-903c2bf-2` via **yarn** or `npm install @magic-sdk/react-native-expo@14.0.1-903c2bf-2 --save` via **npm**.

**Comment by adamjuhasz** [comment]
Thank you for this amazing Friday gift! You Rock!

**Comment by magiclabsFE** [comment]
:rocket: Issue was released in [`@magic-ext/connect@6.2.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/connect%406.2.0), [`@magic-ext/react-native-bare-oauth@8.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/react-native-bare-oauth%408.1.0), [`@magic-ext/react-native-expo-oauth@8.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/react-native-expo-oauth%408.1.0), [`@magic-sdk/react-native-bare@14.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/react-native-bare%4014.1.0), [`@magic-sdk/react-native-expo@14.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/react-native-expo%4014.1.0) :rocket:
